src/shell/commands/data_operations.cpp

```diff
@@ -2397,6 +2397,17 @@ bool count_data(command_executor *e, shell_context *sc, arguments args)
         options.no_value = false;
     else
         options.no_value = true;
+
+    // cuz option only_return_count not return data(hashkey&sortkey&value) to client
+    // all of this options need check data on client side
```

Comment:
I think maybe it's better to explain each of conditions here ?
```suggestion 
    // Decide whether real data should be returned to client. Once the real data is
    // decided not to be returned to client side: option `only_return_count` will be
    // used.
```